educator/src/Dscp/Educator/Web/Server.hs

```diff
@@ -120,9 +123,15 @@ serveEducatorAPIsReal withWitnessApi EducatorWebParams{..} = do
     serveWeb spAddr $
       responseTimeMetric metricsEndpoint $
       educatorCors $
-      serveWithContext (Proxy @EducatorWebAPI) srvCtx $
-         educatorApiServer
-         :<|>
-         studentApiServer
-         :<|>
-         witnessApiServer
+      reify lc $ \logConfigP ->
+      serveWithContext (servedApi logConfigP) srvCtx $
+          educatorApiServer
+          :<|>
+          studentApiServer
+          :<|>
+          witnessApiServer
+      where
+          servedApi
+            :: Reifies config ServantLogConfig
+            => Proxy config -> Proxy (LoggingApi config EducatorWebAPI)
+          servedApi _ = Proxy
```

Comment:
I once tried to do that, and it's not that easy without some boilerplate leftover (due to use of `reflection`).
I wanted to make `logConfigP` be passed via `HasServer` context instead of reflection one day, let me both do this and fulfil your request later.

Reply:
This is the exact copypaste from Witness server.hs. We also have slightly different approach in Faucet server.hs
```
        reify lc $ \(_ :: Proxy lc) ->
        serve (Proxy @(LoggingApi lc FaucetWebAPI)) faucetApiServer
```
I think the Witness one is more explicit.